canvas/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>HTML5 Canvas</title>
+</head>
+<body>
+<canvas id="draw" width="800" height="800"></canvas>
+<script>
+  
+  const canvas = (function(){
```

Comment:
This is weird. This is how to define a function experession
````
const canvas = () => {
   // stuff
    return stuff;
};
```

Reply:
Also, this is super confusing. You've got a function called `canvas` and another variable inside called `canvas`

Reply:
That's a function expression. Immediately invoked. 
changed the name of the other canvas variable.